### PR TITLE
Update API key based validation flow

### PR DESCRIPTION
**Pull request by acoumb**
Current PR aligns with [this](https://github.com/umbraco/Umbraco.Cms.Integrations/pull/145) PR from the CMS package.

### CODE REVIEW

File: src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs [hunk 1]
```diff
@@ -53,14 +92,38 @@ public async Task<bool> Validate(string username, string password, string apiKey
...
                 var user = _userService.GetByUsername(username);
 
-                return user != null && user.Groups.Any(p => p.Alias == _zapierSettings.UserGroupAlias);
+                return user != null && user.Groups.Any(p => p.Alias == _zapierFormsSettings.UserGroupAlias);
             }
 
             return true;
         }
+
+//        public async Task<bool> Validate(string username, string password, string apiKey)
```
Comment (AndyButland) [inline src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs:105]: Should remove if not used.

File: src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs [hunk 2]
```diff
@@ -53,14 +92,38 @@ public async Task<bool> Validate(string username, string password, string apiKey
 
             if (!isUserValid) return false;
 
-            if (!string.IsNullOrEmpty(_zapierSettings.UserGroupAlias))
+            if (!string.IsNullOrEmpty(_zapierFormsSettings.UserGroupAlias))
```
Comment (AndyButland) [inline src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs:95]: This using the "forms" version only - is that right?

Comment (acoumb) [reply src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs:95]: Correct.

File: src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs (hunk 1 above)
Comment (acoumb) [reply src/Umbraco.Forms.Integrations.Automation.Zapier/Services/UserValidationService.cs]: Removed

**Review by AndyButland — APPROVED** [review]

**Pull request merged**